keanu-project/src/main/java/io/improbable/keanu/plating/Plate.java

```diff
@@ -22,6 +24,17 @@ public Plate() {
         this.contents = new HashMap<>();
     }
 
+
+    public <T extends Vertex<?>> void addAll(T... vertices) {
+        addAll(ImmutableList.copyOf(vertices));
+    }
+
+    public <T extends Vertex<?>> void addAll(Collection<T> vertices) {
+        for (T v : vertices) {
+            add(v);
```

Comment:
is there a reason you've used a for loop instead of forEach()?

Reply:
nope - changed